Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListUserControl.ascx.cs

```diff
@@ -392,6 +400,186 @@ private void ChildrenListRemoveRowsButton_Click (object sender, EventArgs e)
+                    {
+                        new ValidatedProperty(lastJob, jobStartDateProperty),
+                    },
+                    "Bad last job date",
+                    "Localized bad last job dates"),
+                new ObjectValidationFailure(
+                    lastJob,
+                    new[]
+                    {
+                        new ValidatedProperty(lastJob, jobStartDateProperty),
+                        new ValidatedProperty(lastJob, jobEndDateProperty)
+                    },
+                    "Bad last job dates",
+                    "Localized bad last job dates"),
+                // Children property failures
+                new PropertyValidationFailure(
+                    person,
+                    childrenProperty,
+                    person.Children,
+                    "Bad children",
+                    "Localized bad children"),
+                // First child failure (visible)
+                new ObjectValidationFailure(
+                    firstChild,
+                    "Bad first child",
+                    "Localized bad first child"),
+                new PropertyValidationFailure(
+                    firstChild,
+                    personPartnerProperty,
+                    firstChild.Partner,
+                    "Bad first child.Partner",
+                    "Localized bad first child.Partner"),
+                new ObjectValidationFailure(
+                    firstChild,
+                    new[]
+                    {
+                        new ValidatedProperty(firstChild, personLastNameProperty),
+                    },
+                    "Bad first child name",
+                    "Localized bad first child names"),
+                new ObjectValidationFailure(
+                    firstChild,
+                    new[]
+                    {
+                        new ValidatedProperty(firstChild, personLastNameProperty),
+                        new ValidatedProperty(firstChild, personFullNameProperty)
+                    },
+                    "Bad first child names",
+                    "Localized bad first child names"),
+                // Last child failure (invisible)
+                new ObjectValidationFailure(
+                    lastChild,
+                    "Bad last child",
+                    "Localized bad last child"),
+                new PropertyValidationFailure(
+                    lastChild,
+                    personPartnerProperty,
+                    lastChild.Partner,
+                    "Bad last child.Partner",
+                    "Localized bad last child.Partner"),
+                new ObjectValidationFailure(
+                    lastChild,
+                    new[]
+                    {
+                        new ValidatedProperty(lastChild, personLastNameProperty),
+                    },
+                    "Bad last child name",
+                    "Localized bad last child names"),
```

Comment:
Here is the answer to the problem we kept debugging. Please check if we have other cases like this. Apart from that the reporting we experienced is correct. As we have a compound validation failure below, some validation failures are (correctly) reported multiple times.

```suggestion
                    "Localized bad last child name"),
```